src/main/java/io/vertx/core/eventbus/ReplyException.java

```diff
@@ -39,6 +39,20 @@ public ReplyException(ReplyFailure failureType, int failureCode, String message,
     this.failureCode = failureCode;
   }
 
+  /**
+   * Create a ReplyException with all attributes, including cause.<br>
+   * Default {@link io.vertx.core.eventbus.impl.codecs.ReplyExceptionMessageCodec} doesn't support Cause!
+   *
+   * @param failureType  the failure type
+   * @param failureCode  the failure code (e.g. 404)
+   * @param message  the failure message
+   */
+  public ReplyException(ReplyFailure failureType, int failureCode, String message, Throwable cause, boolean noStackTrace) {
```

Comment:
This constructor should have `protected` visibility instead of `public` and we shall document that it is meant to be used for extension, together with a custom codec.

Reply:
Fixed